### ISSUE TITLE
MAP ALL not working

### ISSUE THREAD

**Issue by colinmccann** [ixmaps/website2017#66]
Clicking MAP ALL on the Map page will only show one route, because the default is that Multiple TRs is set to Off. Switching the default to On is easiest solution. Otherwise put a conditional on MAP ALL click?

**Comment by Andrew-Clement** [comment]
Yes, this is a bit messy. I asked that the Multiple TRs be off by default as it makes looking at individual routes more difficult. 

I tried to address this in the proposed tooltip:

> Displays all the routes in the current sample of routes below. To see them overlaid, turn on Multiple routes in Settings  

However, I'm now inclined to have the Map All button ignore the the Multiple TR setting and simply overlay all in the current sample (and not change the setting) 

Unless there are  objections, or it would be time consuming to change the behaviour of the Map ALl button, please make it work this way. The tooltip should then simply read:

> Displays all the routes in the current sample of routes below. 

The tooltip for Multiple TRs in Map Settings will need to be adjusted accordingly.

Alternate tooltip wordings are provided in both cases.

**Comment by Andrew-Clement** [comment]
Leave until later. In the meantime, tooltiup wording provides guidance
